ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py

```diff
@@ -0,0 +1,59 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy.functional.backends.torch as ivy_torch
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# allclose
+@handle_cmd_line_args
+@given(
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            ),
+        ),
+        num_arrays=2,
+        min_value=-1.0013580322265625e-05,
```

Comment:
We can actually just remove min_value and max_value paramters in this case as there is no need to constrain them at all :)